README.md

```diff
@@ -1,3 +1,45 @@
 # Token Oracle <img align="right" src="https://github.com/1Hive/website/blob/master/website/static/img/bee.png" height="80px" />
 
-ACL Oracle enforcing token ownership
+The Token Oracle app is an [ACL Oracle](https://hack.aragon.org/docs/acl_IACLOracle). ACL Oracles are small helper functions that plug in to Aragon's access control list (ACL) to do more sophisticated permission evaluation. In the context of Dandelion Orgs, the Token Oracle is used to check if an address holds Dandelion Org tokens and is thus a member of the organization. This is done by setting a minimum balance in the Token Oracle. Then various Dandelion applications can query the Token Oracle to see if an account submitting the intent to perform an action has the minimum balance of tokens required by the Token Oracle. The Token Oracle will return a boolean which can be used to either approve or reject the intent.
+
+#### 🐲 Project stage: development
+
+The Token Oracle app is still in development. If you are interested in contributing please see our open [issues](https://github.com/1hive/time-lock-app/issues).
```

Comment:
Would remove this part as it is already fully developed

Reply:
Good call. Do you think we should remove this section entirely, or update to say something like "completed?"

Reply:
I think it's okay to just remove it.
If we don't explicitly state the project stage it can be assumed as completed. 
Nevertheless if someone wants to check if there are any open issues they can do so in Contributing section